portfolio/src/main/webapp/index.html

```diff
@@ -81,6 +81,9 @@ <h2>More About Me</h2>
       <button id="picture1" class='myButton' onclick="addBabyPhoto()">View Baby Photo</button>
       <button id="picture2" class='myButton' onclick="removeBabyPhoto()">Hide Baby Photo</button>
       <div id="picture-container" style='height: 100%; width: 100%; object-fit: contain'></div>
+      <h2>---End of Week 1---</h2>
+      <body onload="fetchContentFromServlet()">Fetched Content Below:</body>
```

Comment:
It's strange to have a <body> tag in the middle of an html page. Maybe you can move the onload to the top level body, so you don't need another body tag?

Reply:
I tried moving into the header <h2> and changing the "body" to "div" but it doesn't work when I do that

Reply:
Yes, only a few HTML tags support onload attribute, as listed [here: Supported HTML tags](https://www.w3schools.com/tags/ev_onload.asp).

Reply:
What do you mean by moving to the top level body though?

Reply:
Yep. If I understand correctly, there should be only one <body> tag in a html page.